### PR TITLE
fix: get code, code hash and code size from an unregistered address

### PR DESCRIPTION
**Pull request by magicalne**
Handling get_code, get_code_size, and get_code_hash for the unregistered addresses instead of throwing an error.

relate to:
- fix https://github.com/nervosnetwork/godwoken-web3/issues/391

### CODE REVIEW

**Review by Flouse — CHANGES_REQUESTED** [review]
Please fix the conflicts.

File: c/polyjuice_utils.h [hunk 1]
```diff
@@ -129,8 +129,8 @@ int load_script_hash_by_eth_address(gw_context_t *ctx,
   if (_is_zero_hash(script_hash)) {
     return GW_ERROR_NOT_FOUND;
   }
-  ckb_debug("load_script_hash_by_eth_address success");
   return 0;
+  ckb_debug("load_script_hash_by_eth_address success");
 }
```
Comment (Flouse) [inline c/polyjuice_utils.h:134]: Delete this line since it is unreachable.

File: polyjuice-tests/src/test_cases/absent_address.rs [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        block_producer.clone(),
+        0,
+    );
+
+    let contract_account = MockContractInfo::create(&from_eth_address, 0);
+    let new_account_id = state
+        .get_account_id_by_script_hash(&contract_account.script_hash)?
+        .unwrap();
+
+    //call getBalance("0xdB81D2b8154A10C6f25bC2a9225F403D954D0B65")
```
Comment (Flouse) [inline polyjuice-tests/src/test_cases/absent_address.rs:43]:
```suggestion
    // 0xdB81D2b8154A10C6f25bC2a9225F403D954D0B65 is an unregistered eth_address.
    // call getBalance("0xdB81D2b8154A10C6f25bC2a9225F403D954D0B65")
```

File: polyjuice-tests/src/test_cases/absent_address.rs [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    let run_result = generator
+        .execute_transaction(
+            &ChainView::new(&db, tip_block_hash),
+            &state,
+            &block_info,
+            &raw_tx,
+            L2TX_MAX_CYCLES,
+        )
+        .expect("Call getCode");
+    let mut target = [0u8; 64];
```
Comment (Flouse) [inline polyjuice-tests/src/test_cases/absent_address.rs:153]:
```suggestion
    // expected output: { "0": "bytes: 0x"}
    let mut target = [0u8; 64];
```

**Review by Flouse — DISMISSED** [review]

**Review by TheWaWaR — DISMISSED** [review]

**Review by zeroqn — DISMISSED** [review]

**Review by Flouse — APPROVED** [review]

**Pull request merged**